Userland/Utilities/man.cpp

```diff
@@ -78,12 +114,18 @@ int main(int argc, char* argv[])
     auto file = Core::File::construct();
     file->set_filename(make_path(section));
 
+    String pager_command = pager;
+    if (!pager) {
+        pager_command = String::formatted("less -P 'Manual Page {}({}) line %l?e (END):.'", name, section);
```

Comment:
That's not nice, if `name` contains a quote, this is a syntax error, or a borked shell injection path :P

Reply:
Although a man page with that name would have to exist for it to be exploited, I've sanitized single-quotes in the section and name.